README.md

```diff
@@ -29,17 +29,17 @@ Go SDK for the [Auth0](https://auth0.com/) Management API.
 
 ### Requirements
 
-- Go 1.17+
+This library follows the [same support policy as Go](https://go.dev/doc/devel/release#policy). The last two major Go releases are actively supported and compatibility issues will be fixed. While you may find that older versions of Go may work, we will not actively test and fix compatibility issues with these versions.
 
-**go-auth0** tracks [Go's version support policy](https://go.dev/doc/devel/release#policy). 
+- Go 1.19+
 
 ### Installation
 
 ```shell
 go get github.com/auth0/go-auth0
 ```
 
-### Usage
+### Configuring the SDK
```

Comment:
The sample below doesn't just show how to configure the SDK tho, it also shows a modest usage. Maybe we can keep usage?